projects/packages/search/src/dashboard/components/record-meter/index.jsx

```diff
@@ -39,8 +39,16 @@ export default function RecordMeter( {
 		<div className="jp-search-record-meter jp-search-dashboard-wrap" data-testid="record-meter">
 			<div className="jp-search-dashboard-row">
 				<div className="lg-col-span-2 md-col-span-1 sm-col-span-0"></div>
-				<div className="jp-search-record-meter__title lg-col-span-8 md-col-span-6 sm-col-span-4">
-					<h2>{ __( 'Your search records', 'jetpack-search-pkg' ) }</h2>
+				<div className="jp-search-record-meter__content lg-col-span-8 md-col-span-6 sm-col-span-4">
+					<h2>
+						{
+							/* translators: Your Search Index is a breakdown of the site's indexed post type content,
+					such as the number of indexed posts, pages etc. */ __(
+								'Your Search Index',
```

Comment:
The copy guidelines say to use sentence case unless referring to a product name (see p6TEKc-2Lx-p2), so it could be "Your Search index" or "Your search index" but not "Your Search Index".

What's your preference @jeffgolenski?

Reply:
```suggestion
								'Your search index',
```